__tests__/app.test.js

```diff
@@ -91,3 +91,27 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+  describe("/api/articles", () => {
+    it("GET responds with status code 200", () => {
+      return request(app).get("/api/articles").expect(200);
+    });
+    it("GET responds with slug and description fields", () => {
+      return request(app)
+        .get("/api/articles")
+        .then((response) => {
+            expect(response.body.articles.length).toBe(13)
+            response.body.articles.forEach((article) => {
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+              });
+        });
```

Comment:
Need to test for this from the ticket as well

In addition:

the articles should be sorted by date in descending order.
there should not be a body property present on any of the article objects.